scipy/sparse/__init__.py

```diff
@@ -230,7 +230,7 @@
 
 >>> A = lil_array((1000, 1000))
 >>> A[0, :100] = rand(100)
->>> A[1, 100:200] = A[0, :100]
+>>> A[1, 100:200] = A[:, [0]].toarray()[0, :100]
```

Comment:
is this deliberate or has it slipped through the merge commit?

Reply:
When running doctests through the plugin, this example was failing due to a `NotImplementedError` as mentioned here: https://github.com/scipy/scipy/pull/19242#issuecomment-1739410889
Explicit indexing currently solves this error. 
Is there a more effective method you might recommend to address the error?